ui/store/actions.js

```diff
@@ -3273,6 +3273,17 @@ export function setRecoveryPhraseReminderLastShown(lastShown) {
   };
 }
 
+export function setOutdatedBrowserWarningLastShown(lastShown) {
+  return async (dispatch) => {
+    await submitRequestToBackground('setOutdatedBrowserWarningLastShown', [
+      lastShown,
+    ]).catch((err) => {
```

Comment:
Nit: Could you use `try .. catch` here instead of `.catch`? The former will preserve an async stack trace across this line on Chromium-based browsers, leading to better error messages. Generally async/await should be preserved over .then/.catch in all cases for this reason.